### ISSUE TITLE
Images not rendering using renderHtml

### ISSUE THREAD

**Issue by d-karstens** [verbb/vizy#91]
**Description**
If a Vizy field only contains images nothing renders when using `vizyField.renderHtml()`, unless the field also contains some text

We're currently working around this using

```
{% for node in entry.copy.all() %}
  {% if node.type == 'image' %}
    <img src="{{ node.attrs.src }}" alt="{{ node.attrs.alt }}"/>
  {% else %}
    {{ node.renderHtml() }}
  {% endif %}
{% endfor %}
```

**Steps to reproduce**
1. Create a Vizy field with nothing but an image in it
2. Nothing displays when using `{{ vizyField.renderHtml() }}`

**Additional info**
- Plugin version: 1.0.8
- Craft version: 3.7.22
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Thank you, much appreciated!

**Comment by engram-design** [comment]
Fixed in [1.0.9](https://github.com/verbb/vizy/releases/tag/1.0.9)
